src/corelibs/U2View/src/ov_sequence/DetView.cpp

```diff
@@ -120,7 +114,7 @@ DetView::DetView(QWidget *p, SequenceObjectContext *ctx)
     showTranslationAction->setChecked(hasAmino);
 
     assert(ctx->getSequenceObject() != NULL);
-    featureFlags &= !GSLV_FF_SupportsCustomRange;
+    featureFlags &= !GSLV_FF_SupportsCustomRange;    //TODO: bug?
```

Comment:
Why?

Reply:
! is a boolean operation. I think the original intention was to use ~
https://www.geeksforgeeks.org/bitwise-operators-in-c-cpp/

Reply:
Yes, looks like it's true. Fix it, or  create the issue